products/jbrowse-web/src/Loader.tsx

```diff
@@ -282,8 +286,21 @@ const SessionLoader = types
       )
       self.setSessionSnapshot({ ...session, id: shortid() })
     },
+
+    async decodeJsonUrlSession() {
+      // @ts-ignore
+      const session = JSON.parse(self.sessionQuery.replace('json-', ''))
```

Comment:
would want to replace json- just at the beginning, so probably want a regexp here

Reply:
replace doesn't do a replaceAll so should be safe